internal/buildscripts/update-deps

```diff
@@ -33,15 +36,7 @@ for gomod in $( find "$REPO_DIR" -name "go.mod" | grep -v "/examples/" | sort );
         go get go.opentelemetry.io/collector/model@${CORE_VERSION}
     fi
 
-    lines="$( (grep 'github.com/open-telemetry/opentelemetry-collector-contrib/' "$gomod" | grep -v 'ignore-update-deps' | grep -v '^[[:space:]]*//' | grep -v '=>' | grep -v ' // indirect' | sort -u) || true )"
-    for line in $lines; do
-        pkg="$( echo "$line" | awk '{print $1}' )"
-        go get ${pkg}@${CONTRIB_VERSION}
-    done
-
-    IFS=$OFS
-
-    go mod tidy -go=1.16 && go mod tidy -go=1.17  && go mod tidy -go=1.18
+    go mod tidy
```

Comment:
Is the thought here that `go mod tidy` now does everything that the lines being deleted were doing, or is the deleted functionality just no longer necessary?

Reply:
It's no longer necessary now that we've migrated completely to go 1.18.  Also, the